modules/openshift-cluster-maximums-environment.adoc

```diff
@@ -42,15 +42,15 @@
 | m5.2xlarge
 | 8
 | 32
-| gp2
+| gp3
 | 100
 | 3/25/250/500 ^[5]^
 | us-west-2
 
 |===
 [.small]
 --
-1. io1 disks with 3000 IOPS are used for control plane/etcd nodes as etcd is I/O intensive and latency sensitive.
+1. gp3 disks with a baseline performance of 3000 IOPS and 125 MiB/s are used for master/etcd nodes as etcd is latency sensitive. (gp3 volumes do not use burst performance)
```

Comment:
I believe we're using "control plane" instead of "master". 

Suggested revision: `1. gp3 disks with a baseline performance of 3000 IOPS and 125 MB per second are used for control plane/etcd nodes because etcd is latency sensitive. gp3 volumes do not use burst performance.`

Reply:
MiB is correct, I've updated this section with your suggestion